addons/l10n_cl/models/account_move.py

```diff
@@ -109,11 +114,11 @@ def _get_starting_sequence(self):
     def _get_last_sequence_domain(self, relaxed=False):
         where_string, param = super(AccountMove, self)._get_last_sequence_domain(relaxed)
         if self.company_id.country_id == self.env.ref('base.cl') and self.l10n_latam_use_documents:
-            journals = self.journal_id.l10n_cl_sequence_ids.filtered(lambda s: s.l10n_latam_document_type_id == self.l10n_latam_document_type_id).l10n_cl_journal_ids.ids
-            if len(journals) > 1:
-                where_string.replace("journal_id = %(journal_ids)s", "journal_id in %(journal_ids)s")
-                param['journal_ids'] = journals
-            where_string += " AND l10n_latam_document_type_id = %(l10n_latam_document_type_id)s "
+            where_string = where_string.replace('journal_id = %(journal_id)s AND', '')
+            where_string += " AND l10n_latam_document_type_id = %(l10n_latam_document_type_id)s"
+            # FIXME: adding the company_id to where_string raises with unsupported format character
+            # " AND company_id = %(company_id)"
```

Comment:
@suv-bmya Is not it just my fault?  Maybe it should just be %(company_id)s.  I will do a test.

Reply:
uuff! very easy! It was very late yesterday. Yes! You're right! Thank so much!